backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5752520_sys_BusinessRule_M_ProductPrice_UOM_Conversion.sql

```diff
@@ -0,0 +1,130 @@
+WHERE AD_Language='de_CH' AND AD_Message_ID=545529
+;
+
+-- 2025-04-23T12:05:21.617Z
+UPDATE AD_Message base SET MsgText=trl.MsgText, Updated=trl.Updated, UpdatedBy=trl.UpdatedBy FROM AD_Message_Trl trl 
+ WHERE trl.AD_Message_ID=base.AD_Message_ID AND trl.AD_Language='de_CH' AND trl.AD_Language=getBaseLanguage()
+;
+
+-- Value: M_ProductPrice_No_UOLM_Conversion
+-- 2025-04-23T12:05:21.737Z
+UPDATE AD_Message_Trl SET IsTranslated='Y',Updated=TO_TIMESTAMP('2025-04-23 12:05:21.737000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',
+UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Message_ID=545529
+;
+
+-- Value: M_ProductPrice_No_UOLM_Conversion
+-- 2025-04-23T12:05:24.230Z
+UPDATE AD_Message_Trl SET MsgTip='Für die Preiseinheit gibt es noch keine Maßeinheit Umrechnung.',
+Updated=TO_TIMESTAMP('2025-04-23 12:05:24.229000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100
+ WHERE AD_Language='de_CH' AND AD_Message_ID=545529
+;
+
+-- 2025-04-23T12:05:24.231Z
+UPDATE AD_Message base SET MsgTip=trl.MsgTip, Updated=trl.Updated, UpdatedBy=trl.UpdatedBy FROM AD_Message_Trl trl 
+ WHERE trl.AD_Message_ID=base.AD_Message_ID AND trl.AD_Language='de_CH' AND trl.AD_Language=getBaseLanguage()
+;
+
+-- 2025-04-23T12:05:53.112Z
+INSERT INTO AD_BusinessRule (AD_BusinessRule_ID,AD_Client_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,IsActive,IsDebug,Name,Updated,UpdatedBy,Validation_Rule_ID,
+Warning_Message_ID) VALUES (540008,0,0,251,TO_TIMESTAMP('2025-04-23 12:05:53.097000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,
+'Y','N','M_ProductPrice with UOM conversion',TO_TIMESTAMP('2025-04-23 12:05:53.097000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',
+100,540472,540301)
+;
+
+-- 2025-04-23T12:06:12.749Z
+UPDATE AD_BusinessRule SET Warning_Message_ID=545529,
+Updated=TO_TIMESTAMP('2025-04-23 12:06:12.748000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',
+UpdatedBy=100 WHERE AD_BusinessRule_ID=540008
+;
+
+
+
+
+-- Name: M_ProductPrice with UOM Conversion
+-- 2025-04-23T12:25:31.747Z
+INSERT INTO AD_Val_Rule (AD_Client_ID,AD_Org_ID,AD_Val_Rule_ID,Created,CreatedBy,EntityType,IsActive,Name,Type,Updated,UpdatedBy) 
+VALUES (0,0,540709,TO_TIMESTAMP('2025-04-23 12:25:31.743000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',
+100,'D','Y','M_ProductPrice with UOM Conversion','S',TO_TIMESTAMP('2025-04-23 12:25:31.743000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
+;
+
+-- Name: M_ProductPrice with UOM Conversion
+-- 2025-04-23T12:25:39.975Z
+UPDATE AD_Val_Rule SET Code='EXISTS (
+SELECT 1
+FROM M_ProductPrice pp
+         JOIN M_Product p
+              ON pp.M_Product_ID = p.M_Product_ID
+WHERE pp.M_ProductPrice_ID = M_ProductPrice.M_ProductPrice_ID
```

Comment:
why are u selecting again from `M_ProductPrice pp ` when u already have the M_ProductPrice record available?!